Toric/Mathlib/RingTheory/HopfAlgebra/Convolution.lean

```diff
@@ -28,19 +28,42 @@ open scoped ConvolutionProduct RingTheory.LinearMap
 variable {R A C : Type*} [CommSemiring R]
 
 namespace HopfAlgebra
-variable [CommSemiring A] [HopfAlgebra R A]
+
+section Semiring
+
+variable [Semiring A] [HopfAlgebra R A]
+
+lemma _root_.Commute.mul_left_comm {M : Type*} [Monoid M] {a b c : M} (H : Commute a b) :
+    a * (b * c) = b * (a * c) := by rw [← mul_assoc, ← mul_assoc, H]
```

Comment:
That's [`Commute.left_comm`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Group/Commute/Defs.html#Commute.left_comm)